.github/workflows/macos_python.yml

```diff
@@ -8,6 +8,13 @@ on:
   schedule:
     # nightly at 1 AM
     - cron: '0 1 * * *'
+  pull_request:
+    branches:
+    - develop
+    paths:
+      # Run if we modify this yaml file
+      - '.github/workflows/macos_python.yml'
+      # TODO: run if we touch any of the recipes involved in this
```

Comment:
These recipes run for hours, I would not do them on PRs :'(

Reply:
Yeah, the idea behind those is to test routinely at night if a typical data science stack installs.

Reply:
but we could make a few fundamental packages that install not too long in pull request CI... maybe `bzip2` and `openblas`/`py-numpy` or so?